bazel/src/main/java/com/google/api/codegen/bazel/BUILD.bazel.raw_api.mustache

```diff
@@ -32,10 +32,13 @@ java_proto_library(
     deps = [":{{name}}_proto"],
 )
 
-java_grpc_library(
```

Comment:
We can remove the import of `java_grpc_library` above as well.

Reply:
oh great catch thanks.